app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author
```

Comment:
- [ ] Please make sure to refer to the class in order for this association to work exactly like you did in the Post class.